apps/chat/src/components/Marketplace/ApplicationLogs.tsx

```diff
@@ -103,6 +96,21 @@ const LogsFooter = ({ entityId }: LogsFooterProps) => {
           />
         </button>
       </Tooltip>
+      {applicationLogs && (
+        <Tooltip tooltip={t('Download logs')}>
+          <button
+            onClick={() => handleDownload(applicationLogs)}
```

Comment:
you can call `downloadApplicationLogs(applicationLogs)` here. `handleDownload`. handleDownload is redundant I guess